packages/cozy-client/examples/helpers/file.js

```diff
@@ -0,0 +1,116 @@
+const { default: CozyClient, QueryDefinition } = require('cozy-client')
+const fs = require('fs')
+const { FileDocument } = require('cozy-client/dist/types')
+
+const schema = {
+  files: {
+    doctype: 'io.cozy.files'
+  }
+}
+
+const generateFakeFolder = () => ({
+  name: 'folder-name' + Math.random(),
+  icon: 'icon',
+  type: 'directory',
+  dirId: 'io.cozy.files.root-dir'
+})
+
+const generateFakeFile = (folder, i, file) => ({
+  name: `${folder.name}-${i}.txt`,
+  contentType: 'text/plain',
+  icon: 'icon',
+  path: `/${folder.name}/name${i}.txt`,
+  type: 'file',
+  dirId: folder._id,
+  data: file
+})
+
+const createFolder = async client => {
+  const { data: folder } = await client.stackClient
+    .collection('io.cozy.files')
+    .create(generateFakeFolder())
+  return folder
+}
+
+const createFileInsideFolder = async (client, count, folder) => {
+  const readme = fs.readFileSync('../README.md')
+  for (let i = 0; i < count; i++) {
+    let file = generateFakeFile(folder, i, readme)
+    await client.stackClient.collection('io.cozy.files').create(file)
+  }
+}
+/**
+ * Create a folder and add files inside
+ *
+ * @param {CozyClient} client - Cozy client instance
+ * @param {number} count - Number of files to create
+ * @returns {Promise<FileDocument>} Folder created
+ */
+const createFolderWithFilesBulk = async (client, count) => {
+  const folder = await createFolder(client)
+  await createFileInsideFolder(client, count, folder)
+  return folder
+}
+
+/**
+ * Delete a file
+ *
+ * @param {CozyClient} client - Cozy client instance
+ * @param {FileDocument} file - File to delete
+ * @param {string} file._id - Id of the file
+ * @returns {Promise<void>}
+ */
+const deleteFile = async (client, file) => {
+  await client.stackClient
+    .collection('io.cozy.files')
+    .destroy({ _id: file._id })
+}
+
+/**
+ * Delete all the files inside a Cozy does not have path field
+ * Useful to remove orphans
+ *
+ * @param {CozyClient} client - Cozy client instance
+ * @param {boolean} dryRun - if file should be deleted
+ * @returns {Promise<void>}
+ */
+const deleteAllFilesWithoutPath = async (client, dryRun) => {
+  const result = await getAllFiles(client)
+  console.log(result.length, 'files found inside this Drive')
+  const undefinedPathFiles = result.filter(({ path }) => !path)
+  console.log(
+    undefinedPathFiles.length,
+    'files without path found inside this Drive'
+  )
+
+  if (dryRun) {
```

Comment:
I don't get why there is a dry run here? And it seems to be the reverse of what's expected: a dry run it not supposed to actually modify anything

Reply:
ok removed